packages/mermaid/src/docs/vite.config.ts

```diff
@@ -0,0 +1,46 @@
+import { defineConfig, searchForWorkspaceRoot } from 'vite';
```

Comment:
This config is duplicated.
You can add the one in /docs to gitignore as it's only used to generate vitepress and need not be checked in afterwards.

Reply:
I cant remove it.. 
it is necessary to build the docs and is used in the ci/cd to deploy the page.